.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        name: PHPUnit Tests
+        strategy:
+            fail-fast: true
+            matrix:
+                php-versions: [ '7.2', '7.3', '7.4', '8.0', '8.1', 'nightly' ]
+                monolog-versions: [ '1.*', '2.*' ]
+                include:
+                    -   php-versions: '7.1'
+                    -   php-versions: '7.1'
+                        composer-flags: '--prefer-lowest'
+                    -   php-versions: '8.1'
+                        minimum-stability: 'dev'
+                    -   php-versions: 'nightly'
+                        minimum-stability: 'dev'
+        continue-on-error: ${{ matrix.php-versions > '8.0' }}
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php-versions }}
+                    ini-values: zend.exception_ignore_args=false
+                    tools: flex
+
+            -   name: Get composer cache directory
+                id: composer-cache
+                run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            -   name: Cache dependencies
+                uses: actions/cache@v2
+                with:
+                    path: ${{ steps.composer-cache.outputs.dir }}
+                    key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+                    restore-keys: ${{ runner.os }}-composer-
+
+            -   name: Set composer minimum stability to dev
```

Comment:
Would be better described as `Configure composer minimum stability` as it supports setting it to something else than `dev`

Reply:
Done